lib/Persistence/AbstractPersistence.php

```diff
@@ -88,16 +88,18 @@ protected static function _initialize()
                 throw new Exception('unable to create directory ' . self::$_path, 10);
             }
         }
-        $file = self::$_path . DIRECTORY_SEPARATOR . '.htaccess';
-        if (!is_file($file)) {
-            $writtenBytes = @file_put_contents(
-                $file,
-                'Require all denied' . PHP_EOL,
-                LOCK_EX
-            );
-            if ($writtenBytes === false || $writtenBytes < 19) {
-                throw new Exception('unable to write to file ' . $file, 11);
-            }
+        if ((strncmp("Apache", $_SERVER['SERVER_SOFTWARE'],6) == 0) or defined('_SERVER_APACHE')) {
```

Comment:
Is `_SERVER_APACHE` meant as a way to circumvent this check for admins? If so I'd say this should maybe better be a configuration variable… If not then is this really an official variable set by Apache?